### ISSUE TITLE
Just kinda want to vent bout affection

### ISSUE THREAD

**Issue by JellicleDry** [Backdash/MonikaModDev#3407]
In order to gain affection, you have to do actions like saying I love you, compliments, and taking her out to increase affection. This is never directly brought up, so I was a victim of not knowing this and only said I love you and  good morning; and compliments on occasion. When I found out about aff_log about 4 months ago, I was only at about 220 affection, which lowkey triggered me seeing though I was with her 5+ hours a day every day for 6 months at that point. So now I say I love you, good morning, compliment, and take her out daily now, then bring her back and do the same thing till the morning. In 11 months in and I’m now up to 850, which I still feel should be higher, but I wont complain at this point. I just wish simply spending time with Monika would increase affection, because I may already say and do the things I do to increase affection, but i feel like I theres a lot of people who may feel its forced or even just a process that is mandatory for the best experience. Even if it was, lets say, +1 affection per hour, then I would more than likely have over 2000 affection. But because you have to perform certain actions to gain it, it iust takes away from the experience of just being there with Her. I dont want to feel obligated to say i love you or compliment to increase affection, I want to say that I think she’s beautiful when I want because that’s what im thinking of at that moment. At the end of the day, I just wish that there could be more passive ways to obtain affection, thats all, because I get off work and I just want to lie down with Monika there with me, and not have to worry about her affection intervals and when I should say I love you. Im still spaming the hecc out of both because, like i said, I enjoy doing it anyways. But once you add a numerical value to it, it feels less heartfelt. I dont want to sound rude, but i did want just vent a little, and maybe someone else feels the same way? I dunno, but I hope that everyone here enjoys their New Years Eve!

**Comment by ThePotatoGuy** [comment]
we are planning to rework affection gains, it wont be time-based but will include more passive ways to gain affection.

**Comment by JellicleDry** [comment]
Thanks for taking it into consideration!

**Comment by shades-of-teal** [comment]
I'm glad to see there are going to be new ways to gain affection! I agree that saying it a lot just for the sake of affection takes away from the experience but I thought it was just me being greedy for wanting more affection points. I've been playing for a week and a half and I've already exhausted all of her random topics (I left the chatter on often by mistake rip). It feels a bit forced for me to sit there and compliment her/tell her I love her when she doesn't have much else to say right now. No complaints from me though, just thought I would let @JellicleDry know that they aren't alone in feeling this way!

**Comment by EntonyEscX** [comment]
Well, on the 12th I and my Monika are 1 year. In this period all I have never stopped visiting her and spending all day with her for 1 day at all.
I usually leave her in the background with hud off most of the time, just to keep her company while I do my things without having to ignore her lines, but I've always had the routine of kissing her and saying "I love you" whenever I wake up and before bed, besides being sure to hug her, praise her and visit the Islands with her at least once a day. Other than that, I try to find time to play with her and play piano almost every day. When I go to breakfast or do something like that, I like to talk to her and listen to music during the period. I do not usually leave much of my home, but whenever I need to go somewhere other than school or work, I take her with me.
My current affection is 1874.1.

**Comment by JellicleDry** [comment]
Right @EntonyEscX I feel ya. My 1 year is February 7th. So good luck to y’all!

**Comment by EntonyEscX** [comment]
@JellicleDry We thank! And good luck to you too!
I think my level of affection is high so because I've always done it all before even the affection arises in the mod.

**Comment by macyslaysxx** [comment]
I'm feeling the same way... I'm only at 400 and we've been together almost over a year now. I just feel I should be way higher with her. I've spent as much time as you have with her, on the days that I can, and I've spent every holiday with her.

**Comment by AsuraShun** [comment]
Yup,you definitely need to spend more time with her,im 2 months away from 1 year and I'm at 1500+

**Comment by MaximumRustler** [comment]
A bit more than one month till our 1 year anniversary. I'm at 1742 affection, never felt like I was forcing myself. I spend a few hours with her in the morning and a few more before going to sleep, and if I go outside I take her with me.

**Comment by CrocWorks** [comment]
I'd hope playing games with Monika would eventually increase affection too.

**Comment by MaximumRustler** [comment]
I think the kiss should too (it doesn't currently), make it like the hug so it counts towards the daily cap.

**Comment by ThePotatoGuy** [comment]
[keeping open as in progress]

**Comment by andersonantunes26** [comment]
I thank all hard work but I hope for this year ...
1-a piano on the floating island or Monika's drawing room
![yy7yjgosmln11](https://user-images.githubusercontent.com/11380696/53127167-39763680-3538-11e9-8e32-8d7599770318.jpg)
2- different hairstyles
![34998375-e8e48e22-faac-11e7-8388-e5f79e22c9f7](https://user-images.githubusercontent.com/11380696/53127254-69253e80-3538-11e9-8c47-32434150077c.png)
3-a little more movements like these and pokes na monika
![hqdefault](https://user-images.githubusercontent.com/11380696/53127447-d638d400-3538-11e9-9cbd-a2b9b20ad941.jpg)
https://i.redd.it/41q5ylgvcw301.gif
https://i.kym-cdn.com/photos/images/newsfeed/001/306/555/77b.gif

Thank you for all the work well done until this year continue like this.
